modules/mobile/docs/openapi.yaml

```diff
@@ -1467,7 +1467,7 @@ paths:
             application/json:
               schema:
                 $ref: "./schemas/Errors.yml"
-  /v0/claims-and-appeals/overview:
+  /v0/claims-and-appeals-overview:
```

Comment:
"fixing docs" seems like a different concept than fixing the parallel gem.  Those should probably be two separate PRs.  Maybe the change is small enough the reviewer will allow it, but in the future please try to have one concept per PR

Reply:
Gotcha, will do in the future